src/validators/url.py

```diff
@@ -1,8 +1,8 @@
 """URL."""
 
 # standard
-from functools import lru_cache
 import re
+from functools import lru_cache
```

Comment:
Please do not re-order this. [PyCQA](https://github.com/python-validators/validators/blob/master/.github/workflows/pycqa.yaml) workflow will fail. 

I know it does not look good, but currently ruff does not support line-length based import sorting. See https://github.com/astral-sh/ruff/issues/3119.